src/components/DocumentationTopic/TopicsTable.vue

```diff
@@ -110,8 +110,20 @@ export default {
     sectionsWithTopics() {
       return this.sections.map(section => ({
         ...section,
-        topics: section.identifiers.reduce(
-          (list, id) => (this.references[id] ? list.concat(this.references[id]) : list),
+        topics: section.identifierItems.reduce(
+          (list, item) => {
+            if (this.references[item.identifier]) {
+              const ref = this.references[item.identifier];
+              if (item.overrideTitle) {
+                ref.title = item.overrideTitle;
```

Comment:
Because JS objects are retrieved as references, this will change the source object inside the `this.references` entirely, overriding the original title and content everywhere, which I think is not good.

We should create a clone of that reference, so we can safely augment things as needed.